lib/src/util/css_value_util.dart

```diff
@@ -132,7 +132,7 @@ class CssValue implements Comparable<CssValue> {
 
   /// Returns whether this value's [number] and [unit] are equal to that of [other].
   @override
-  bool operator ==(dynamic other) {
+  bool operator ==(Object other) {
```

Comment:
This fixes a new built-in Dart warning; apparently `==` will never be passed a null value (`== null` bypasses `operator==`).

Reply:
interesting